install-dev/init.php

```diff
@@ -70,6 +70,7 @@
             )
         );
     } else {
+        require_once __DIR__ . DIRECTORY_SEPARATOR . 'classes/datas.php';
```

Comment:
Should be done in the upgrade.php script instead.

Reply:
@PierreRambaud Done